CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * [#7193](https://github.com/rubocop-hq/rubocop/issues/7193): Prevent `Style/PercentLiteralDelimiters` from changing `%i` literals that contain escaped delimiters. ([@buehmann][])
 * [#7590](https://github.com/rubocop-hq/rubocop/issues/7590): Fix an error for `Layout/SpaceBeforeBlockBraces` when using with `EnforcedStyle: line_count_based` of `Style/BlockDelimiters` cop. ([@koic][])
 * [#7569](https://github.com/rubocop-hq/rubocop/issues/7569): Make `Style/YodaCondition` accept `__FILE__ == $0`. ([@koic][])
+* [#7291](https://github.com/rubocop-hq/rubocop/issues/7291): Fix misleading error messages in `Layout/AlignHash` cop to specify how to align the hash. ([@jsartin513][])
```

Comment:
I think changelog entry is unnecessary because this update does not change a behavior of the `Layout/HashAlignment` cop feature.